lib/bridge/IrcHandler.js

```diff
@@ -409,11 +409,13 @@ IrcHandler.prototype.onMode = Promise.coroutine(function*(req, server, channel,
         return;
     }
 
+    if (mode === "s") {
+        // Update the visibility for all rooms connected to this channel
+        return this.ircBridge.updateVisibilityMap(true, server.domain + ' ' + channel, enabled);
```

Comment:
How does this not crash at runtime? The method is `_updateVisibilityMap` not `updateVisibilityMap`.